edx-platform/pearson-amazon-theme/lms/templates/student_account/register.underscore

```diff
@@ -71,7 +71,15 @@
         </button>
     </div>
 </form>
-<script>
-    const fullname_field_label = document.querySelector('label[for="register-name"]');
-    fullname_field_label.textContent = "Company Name"
+<script type="text/javascript" id="overrideFieldNames">
```

Comment:
What's the purpose of the id tag? I don't think you are using it somehow. @Nekenhei